src/popover/types.js

```diff
@@ -80,6 +80,9 @@ export type BasePopoverPropsT = {
   mountNode?: HTMLElement,
   /** How long should be fade out animation in ms, default 0ms */
   animateOutTime?: number,
+  /** Popper options override */
```

Comment:
can we add a link to their docs, so everyone will know what options are available?